fetch/api/request/request-consume-empty.html

```diff
@@ -61,20 +61,32 @@
      });
     }
 
-    function checkRequestWithNoBody(bodyType, checkFunction) {
+    function checkBodyFormDataError(request) {
+      return request.formData().then(
+        function(bodyAsFormData) {
+          assert_unreached("FormData parsing should fail");
+        },
+        function(e) {
+          assert_throws(new TypeError(), function() { throw e; });
```

Comment:
To check that the exception thrown by formData (e) is a TypeError exception.
I replaced this with a promise_rejects test as instructed by annevk.